.openshiftio/application.yaml

```diff
@@ -20,6 +20,10 @@ parameters:
     displayName: Source Directory
     value: .
     required: true
+  - name: RELEASE_VERSION
+    description: The location within the source repo of the application
+    displayName: release version
```

Comment:
could you also add a default release version since the launcher doesn't know about this parameter.  i think the other boosters that were updated have a default value of whatever the package.json version is

Reply:
Sure!

Reply:
done https://github.com/nodeshift-starters/nodejs-rest-http/pull/120/commits/1161c20e22f776c2477635ec327b4c8944563652